website/project/model.py

```diff
@@ -3580,6 +3629,17 @@ def _on_reject(self, user, token):
         )
 
 
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
```

Comment:
The base class raises NotImplementedErrors if these methods are not overridden. Typically Sanction subclasses have some sort of automatic behavior upon approval (send an email, retract a registration, etc).

In the case of the DraftRegistration extra user action is necessary before any further action is taken, so _on_complete and _on_reject (at least for now) should do nothing.